src/esp32/esp32_bt_gattc.c

```diff
@@ -36,6 +36,14 @@
 
 static esp_gatt_if_t s_gattc_if = 0;
 
+static esp_bt_uuid_t notify_descr_uuid = {
+  .len = ESP_UUID_LEN_16,
+  .uuid = {.uuid16 = ESP_GATT_UUID_CHAR_CLIENT_CONFIG,},
+};
+
+// TODO this is hacky - using gloval variable
```

Comment:
yeah, this is really hacky :)
let's at least save conn_id of the connection that asked for notifications, so it is at least a little bit less likely to break.
because currently if you have conn 1, and then conn 2, it is basically impossible to enable notifications on conn 1.